src/Concur/Core/Types.purs

```diff
@@ -99,7 +99,8 @@ instance bindWidget :: Bind (Widget v) where
         case res of
           View v -> cb (View v)
           Completed a -> do
-            _ <- join (Ref.read cancelerRef)
+            cA <- join (Ref.read cancelerRef)
+            void $ runCallback cA \_ -> pure unit
```

Comment:
@ajnsit This line is here to actually run the A canceller before moving onto widget B. So `orr` gathers up all the cancellers and the bind runs them. This is important when we have competing Aff Widgets.   Run the Timers example in purescript-concur-react with/without this commit to see what I mean.